### DISCUSSION TITLE
Using a Logitech web cam mic for Jamulus

### DISCUSSION THREAD

**Discussion by OttoShreek** [jamulussoftware/jamulus#1399] [Technical Support & Help]
I am working on getting a choir set up to use Jamulus for virtual rehearsal.  Choir members have all kinds of setups that _might_ work for Jamulus.  One lady has a Windows 10 laptop with a Logitech web cam that she uses for Zoom.  The mic is available  in the Windows 10 Settings for Sound, and tests OK using Voice Recorder and Zoom.  However, when trying to configure ASIO4ALL I get "device status: Unavailable - In use by another application".  There is no other application running.  Jamulus runs OK, and she can hear other clients, but of course cannot add her voice.

Has anyone succeeded in using a Logitech web cam mic with Jamulus, or do I ask her to find something else to use?

**Comment by hoffie** [comment]
> "device status: Unavailable - In use by another application".

This sounds like a general Windows/ASIO4ALL issue to me, not necessarily related to the specific hardware. Some things to try:
- Try the latest ASIO4ALL version (2.15 beta)
- Try disabling/stopping anything which may touch Audio. Steam, Discord, Teams, WebEx, Cortana, ...
- Try disabling exclusive microphone access in Windows (although I think this may have negative consequences on latency and/or quality)
